integration-cli/docker_cli_daemon_test.go

```diff
@@ -1804,3 +1804,44 @@ func (s *DockerDaemonSuite) TestDaemonStartWithDefalutTlsHost(c *check.C) {
 		c.Fatalf("docker version should return information of server side")
 	}
 }
+
+// Test Daemon starts when the repositories-aufs file is removed
+func (s *DockerDaemonSuite) TestDaemonStartWithNoRepoConf(c *check.C) {
```

Comment:
@LK4D4 Not sure if I fully got your suggestion. imu depending on the storage driver being used, the repo file which get created as repositories-{aufs,vfs,..}, therefore have used `s.d.storageDriver` instead of hard-code the values. Instead should I update the comment line as repositories-{*} ?